### DISCUSSION TITLE
Welcome to stremio-addon-debrid-search Discussions!

### DISCUSSION THREAD

**Discussion by MrMonkey42** [MrMonkey42/stremio-addon-debrid-search#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mahmoud23go** [comment]
Re: [danamag/stremio-addons-list] http://127.0.0.1:11470/stremio://stremio-addons.netlify.app/catalog.json (Issue #270) (رد: [danamag/stremio-addons-list] http://127.0.0.1:11470/stremio://stremio-addons.netlify.app/catalog.json (الإصدار رقم 270)).  لا اعرف ماذا افعل أريد المساعده في تعديل هذا

**Comment by bequbed** [comment]
Is the add on working for everyone? It's not loading for me. I uninstalled and installed it again but no luck.

**Reply by MrMonkey42** [reply]
Just checked. Working fine for me

**Reply by bequbed** [reply]
I tried few times to uninstall and install again but for some reason it just does not show any files.  I know that the files are actually on RD since I have checking that through Debrid Media Manager website. 
It was working fine until Saturday last week. 

![image](https://github.com/MrMonkey42/stremio-addon-debrid-search/assets/107822319/e1bcfd31-6d57-4121-ac28-caf24bc6336b)
![image](https://github.com/MrMonkey42/stremio-addon-debrid-search/assets/107822319/8e15bc7b-384b-4b54-ad35-b69e41652e14)

**Reply by bequbed** [reply]
Any suggestions? I just can't figure out why it's not loading on stremio.

**Reply by flo1043** [reply]
I have the same way, with an account it's ok but with others (1 RD and 1 AD) it doesn't work. I got a message saying it failed to load. I installed uninstalled several times, I tested with a working account and it works regardless of the stremio account used and the other RD or AD accounts do not work regardless of the account. The parameters are identical whether it is stremio or RD. Thank's for the help :-)

**Reply by flo1043** [reply]
It's been working since that day. I have debridsearch results which are functional in searching for a film or series that I added to my debridsearch (RD). Thank you again for this great application
